### PR TITLE
Front end customer

### PR DESCRIPTION
**Pull request by FillipKarlsson**

### CODE REVIEW

File: Project/public/javascripts/choose-create-list-frontEnd.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        var dropdowns = document.getElementsByClassName("myLists");
+        var i;
+        for (i = 0; i < dropdowns.length; i++) {
+            var openDropdown = dropdowns[i];
+            if (openDropdown.classList.contains('show')) {
+                openDropdown.classList.remove('show');
+            }
+        }
+    }
+}
```
Comment (Tetre2) [inline Project/public/javascripts/choose-create-list-frontEnd.js:19]: i have not looked in to it but cant you "pre load" the dropdown when a user loades the page and thus remove the need for this kinda crazy function

File: Project/public/javascripts/choose-create-list-frontEnd.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+});
+
+function createListButtons(lists){
+    for (x in lists){
+
+        var linkButton = document.createElement("a");
+        linkButton.innerHTML = lists[x].name;
+
+        var code = 'return setLocalStorage('+lists[x].id+')'
+        linkButton.setAttribute('onClick', code);
```
Comment (Tetre2) [inline Project/public/javascripts/choose-create-list-frontEnd.js:34]: why the weard "onClick" function?

File: Project/public/javascripts/delivery-information-frontEnd.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            return localLists[i];
+        }
+    }
+    console.error("no list with that id");
+}
+
+
+function notifyUser() {
+
+}
```
Comment (Tetre2) [inline Project/public/javascripts/delivery-information-frontEnd.js:49]: unused function. remove

File: Project/public/stylesheets/choose-create-list.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    background-color: rgb(136, 136, 136);
+    color: black;
+    padding: 16px;
+    width: 250px;
+    height: 100px;
+    font-size: 25px;
+    text-align: left;
+    border: none;
+    cursor: pointer;
+    position: absolute;
```
Comment (Tetre2) [inline Project/public/stylesheets/choose-create-list.css:11]: please refrain from using position absolute, it makes it harder to make the page dynamic

File: Project/public/stylesheets/choose-create-list.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+.myLists a {
+    color: black;
+    padding: 20px 60px;
+    text-decoration: none;
+    display: block;
+}
+
+.show {display: block;}
+
+.ycenter {
```
Comment (Tetre2) [inline Project/public/stylesheets/choose-create-list.css:45]: ycenter is not a good class name

File: Project/public/javascripts/choose-create-list-frontEnd.js (hunk 2 above)
Comment (KarlMurgard) [reply Project/public/javascripts/choose-create-list-frontEnd.js:34]: We haven't done this part, it's just copied from develop and moved it to a new javascript file

File: Project/public/javascripts/delivery-information-frontEnd.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
+
+//Global variable
+var localLists;
+retriveShoppinglistsFromBackend();
+
+function publishList() {
+    sendDataToBackEnd();
+    notifyUser();
```
Comment (Eyzic) [inline Project/public/javascripts/delivery-information-frontEnd.js:8]: This method does not seem to exist

File: Project/public/logIn.html [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Logga in</title>
+    <link rel="stylesheet" type="text/css" href="stylesheets/style.css">
+</head>
+
+<body>
+        <h1 class="xPos">HELPER</h1><br><br><br>
```
Comment (Eyzic) [inline Project/public/logIn.html:11]: remove <br>

File: Project/public/logIn.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```
Comment (Eyzic) [inline Project/public/logIn.html:1]: Is this html page even used?

File: Project/public/order-preferences.html [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="UTF-8">
+        <title>Ange orderinformation</title>
+        <script type="text/javascript" src="javascripts/order-preferences-frontEnd.js"></script>
+        <link rel="stylesheet" type="text/css" href="stylesheets/order-preferences.css">
+        <link rel="stylesheet" type="text/css" href="stylesheets/style.css">
+        
+    </head>
+
+    <body>
```
Comment (Eyzic) [inline Project/public/order-preferences.html:13]: put stuff in divs or section tags.

File: Project/public/publishList.html [hunk 10]
```diff
@@ -2,13 +2,14 @@
 <html lang="en">
 	<head>
 		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<link rel="stylesheet" type="text/css" href="stylesheets/main.css">
 		<title>Helper-shopper</title>
 	</head>
 	<body>
     <nav>
```
Comment (Eyzic) [inline Project/public/publishList.html:10]: is this page used?

File: Project/public/stylesheets/choose-create-list.css [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+    color: black;
+    padding: 16px;
+    width: 250px;
+    height: 100px;
+    font-size: 25px;
+    text-align: left;
+    border: none;
+    cursor: pointer;
+    left: 50%;
+    -ms-transform: translate(-50%);
```
Comment (Eyzic) [inline Project/public/stylesheets/choose-create-list.css:12]: would prefer to not use transform.

File: Project/public/stylesheets/create-list.css [hunk 12]
```diff
@@ -0,0 +1,89 @@
+.smallButton {
+    width: 15%;
+    height: 7%;
+    font-size: 10px;
+}
```
Comment (Eyzic) [inline Project/public/stylesheets/create-list.css:5]: check for classes not used

File: Project/public/stylesheets/customer-shopper.css [hunk 13]
```diff
@@ -0,0 +1,25 @@
+.ytopcenter {
+    position: absolute;
+    top: 10%;
+    -ms-transform: translate(-50%);
```
Comment (Eyzic) [inline Project/public/stylesheets/customer-shopper.css:4]: neight of ytopcenter and ybottomcenter was used in html as far as I could tell. If that is the case then remove this styling classes.

File: Project/public/stylesheets/customer-shopper.css [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+  .menuButton {
+    background-color: gray;
+    margin-left: 25px;
+    margin-top: 13px;
+    width: 43px;
+    text-align: center;
+    line-height: 47px;
+    font-size: 39px;
+    border-radius: 30px;
+    vertical-align: top;
```
Comment (Eyzic) [inline Project/public/stylesheets/customer-shopper.css:24]: be aware that this does nothing on inline elements.

File: Project/public/stylesheets/customer-start.css [hunk 15]
```diff
@@ -0,0 +1,20 @@
+.ytopcenter {
+    position: absolute;
+    top: 10%;
+    -ms-transform: translate(-50%);
+    transform: translate(-50%);
+  }
+
+  .ybottomcenter {
```
Comment (Eyzic) [inline Project/public/stylesheets/customer-start.css:8]: same as above comment

File: Project/public/stylesheets/delivery-information.css [hunk 16]
```diff
@@ -0,0 +1,23 @@
+#adress{
+    top: 22%;
```
Comment (Eyzic) [inline Project/public/stylesheets/delivery-information.css:2]: none of these seems to be used.

File: Project/public/stylesheets/order-preferences.css [hunk 17]
```diff
@@ -0,0 +1,122 @@
+.textSize {
+    font-size: 120%;
```
Comment (Eyzic) [inline Project/public/stylesheets/order-preferences.css:2]: a lot of code that is copy and pasted. Look if you cant find an abstraction that works on multiple elements.

File: Project/public/stylesheets/style.css [hunk 18]
```diff
@@ -1,8 +1,87 @@
...
 
 a {
-  color: #00B7FF;
+  text-decoration: none;
+  color: black;
+}
+.xPos {
+  position: absolute;
+  left: 50%;
+  -ms-transform: translate(-50%);
```
Comment (Eyzic) [inline Project/public/stylesheets/style.css:18]: again, would avoid transform.

**Pull request merged**